libs/vscode/configuration/src/lib/configuration-keys.ts

```diff
@@ -9,6 +9,7 @@ export const GLOBAL_CONFIG_KEYS = [
   'enableTaskExecutionDryRunOnChange',
   'projectViewingStyle',
   'moveGeneratorPatterns',
+  'useNewGenerateUIPreview',
```

Comment:
Does the case matter here now that the package.json is using `useNewGenerateUiPreview`?